harper-core/tests/text/linters/Alice's Adventures in Wonderland.snap.yml

```diff
@@ -594,6 +617,16 @@ Message: |
 
 
 
+Lint:    Miscellaneous (0 priority)
+Message: |
+     317 | “Perhaps it doesn’t understand English,” thought Alice; “I daresay it’s a French
+         |                                ^~~~~~~ Consider adding an article before this noun.
```

Comment:
Oh I have probably not yet annotated all of the names of languages as mass nouns. Many are both mass and countable like "German" but not "English". (Linguists do talk about Englishes but non-linguists don't)